benchmarks/src/bin/tpch.rs

```diff
@@ -1140,6 +1140,7 @@ mod tests {
         test_round_trip!(q3, 3);
         test_round_trip!(q5, 5);
         test_round_trip!(q6, 6);
+        test_round_trip!(q7, 7);
```

Comment:
What's the status of query 8,9,13,14,19 in Ballista?
Those are now included in the DataFusion CI

Reply:
They are all broken in ballista due to different ser/de errors, my plan is to fix them one at a time so we are on par with datafusion.